src/apify_shared/consts.py

```diff
@@ -54,47 +54,60 @@ class ActorEventTypes(str, Enum):
     ABORTING = 'aborting'
 
 
+class ActorEnvVars(str, Enum):
+    """Possible Apify-specific environment variables prefixed with "ACTOR_"."""
+
+    # TODO: document these
```

Comment:
What is meant by this TODO? The class docstring is there, so maybe it could be removed?

Reply:
It means we should describe the meaning of each environment variable in the enum. E.g. see `MetaOrigin` class.

Reply:
Ok, thanks